packages/pn-pa-webapp/src/pages/ToSAcceptance.page.tsx

```diff
@@ -46,18 +44,25 @@ const TermsOfService = ({ isFirstAccept, consentVersion }: TermsOfServiceProps)
   );
 
   const handleAccept = () => {
-    void dispatch(acceptToS(consentVersion))
+    void dispatch(acceptToS(tosConsent.consentVersion))
```

Comment:
I should use the async/await logic.
I have a doubt, if tos or privacy are already accepted we can avoid to redo both api calls?

Reply:
Done.